.github/workflows/bahmni_metabase_build_publish.yaml

```diff
@@ -35,4 +37,38 @@ jobs:
           context: package/docker/
           no-cache: true
           push: true
-          tags: bahmni/bahmni-metabase:${{env.ARTIFACT_VERSION}},bahmni/bahmni-metabase:latest
\ No newline at end of file
+          tags: bahmni/bahmni-metabase:${{env.ARTIFACT_VERSION}},bahmni/bahmni-metabase:latest
+
+      - name: Update Version and Image Tag
+        run: |
+          yq --inplace '.image.tag = "${{ env.ARTIFACT_VERSION }}"' $HELM_CHART_PATH/values.yaml
```

Comment:
values.yaml file is not present in the path (package/helm)

Reply:
@pajmera4 I guess you are checking in MAIN branch. These changes are in BAH-2665.

Reply:
No I was looking into the same PR. There was a confusion, values.yaml is present in the same dir

Reply:
Okay. My bad there.